mod/wikirate_source/set/type/source/source.rb

```diff
@@ -88,12 +85,30 @@ def has_file_or_text?
   
 end
 
+def is_url? url
+  url.start_with?("http://") || url.start_with?("https://")
+end
+
+def is_wikirate_url? url
```

Comment:
Use def with parentheses when there are parameters.<br>Rename `is_wikirate_url?` to `wikirate_url?`.